apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -237,7 +237,7 @@ add_missing_to_genesis_block(Val) ->
     Val.
 
 handle_ping(#{<<"source">> := Src} = PingObj) ->
-    %% Source is only contains host and port
```

Comment:
You may be squashing changes addressing comments, and that are not automatically generated, in commit meant for generated code changes. It may be easier to just squash as one commit.